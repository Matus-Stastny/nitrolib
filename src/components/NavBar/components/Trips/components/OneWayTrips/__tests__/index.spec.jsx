// @flow strict
import * as React from "react";
import { shallow } from "enzyme";

import { OneWayTripsUnwrapped as OneWayTrips } from "..";

const item: any = {
  __typename: "BookingOneWay",
  destinationImageUrl: "url",
  databaseId: "Qm9va2luZ09uZVdheTozNzA1ODU5",
  passengerCount: 2,
  trip: {
    arrival: {
      airport: {
        city: {
          name: "Moscow",
        },
      },
      localTime: "2017-08-03T23:45:00.000Z",
    },
    departure: {
      airport: {
        city: {
          name: "Prague",
        },
      },
      localTime: "2017-08-03T21:10:00.000Z",
    },
  },
};

describe("#OneWayTrips", () => {
  test("render", () => {
    const wrapper = shallow(<OneWayTrips item={item} onSelect={jest.fn()} />);

    expect(wrapper).toMatchSnapshot();
  });
});
