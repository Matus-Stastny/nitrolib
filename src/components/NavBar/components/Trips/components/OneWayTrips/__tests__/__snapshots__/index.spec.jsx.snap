// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#OneWayTrips render 1`] = `
<TripItem
  arrivalCity="Moscow"
  arrivalTime="2017-08-03T23:45:00.000Z"
  countOtherCities={0}
  departureCity="Prague"
  departureTime="2017-08-03T21:10:00.000Z"
  id="Qm9va2luZ09uZVdheTozNzA1ODU5"
  img="url"
  lang="en"
  passengerCount={2}
/>
`;
